eox_nelp/course_experience/api/v1/tests/test_views.py

```diff
@@ -61,20 +61,7 @@ def setUp(self):
                     "status": self.my_unit_like.status,
                     "item_id": f"{self.my_unit_like.item_id}",
                 },
-                "relationships": {
-                    "author": {
-                        "data": {
-                            "type": "User",
-                            "id": f"{self.user.id}",
-                        }
-                    },
-                    "course_id": {
-                        "data": {
-                            "type": "CourseOverview",
-                            "id": f"{self.my_course.id}",
-                        }
-                    }
-                }
+                "relationships": self.make_relationships_data()
```

Comment:
where are you testing this new behavior ?

Reply:
https://github.com/eduNEXT/eox-nelp/blob/9bd8f11c998c7f17d2c09da37cf0ac2173c8322e/eox_nelp/course_experience/api/v1/tests/mixins_helpers.py#L233
https://github.com/eduNEXT/eox-nelp/blob/9bd8f11c998c7f17d2c09da37cf0ac2173c8322e/eox_nelp/course_experience/api/v1/tests/mixins_helpers.py#L336